kubeai/models/llama-3.1-8b-instruct-gaudi.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Source: models/templates/models.yaml
+apiVersion: kubeai.org/v1
+kind: Model
+metadata:
+  name: llama-3.1-8b-instruct-gaudi
+spec:
+  features: [TextGeneration]
+  url: hf://meta-llama/Meta-Llama-3.1-8B-Instruct
+  cacheProfile: nfs
+  engine: VLLM
+  args:
+    - --tensor-parallel-size=1
+    - --block-size=128
+    - --max-num-seqs=256
+    - --max-seq_len-to-capture=2048
```

Comment:
hyphens and dash in single name .. any typo here?

Reply:
no typos, it is the way to do it here.